spec/spec_helper.rb

```diff
@@ -49,6 +49,14 @@ def create_challenge(attrs = {})
   Challenge.create!(user:, agency:, title:, challenge_manager_users:)
 end
 
+def create_evaluation_form(attrs = {})
+  title = attrs[:title] || "test challenge"
+  challenge_id = attrs[:challenge_id] || Challenge.first.id
```

Comment:
this only works if there is already a challenge created in the DB. can you use the create_challenge() helper instead of Challenge.first?
```suggestion
  challenge_id = attrs[:challenge_id] || create_challenge.id
```